tests/test_allocations.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+from owca.allocators import _calculate_task_allocations, _calculate_tasks_allocations, \
+    RDTAllocation, AllocationType, convert_tasks_allocations_to_metrics, \
+    _parse_schemata_file_domains, _count_enabled_bits, _merge_rdt_allocation
+from owca.metrics import Metric, MetricType
+
+r = AllocationType.RDT
```

Comment:
```suggestion
RDT = AllocationType.RDT
```
(I completely missed this alias below, using full name would increase readability).

Reply:
fixed - I will use "rdt" without need to put `AllocationType.RDT` in fixtures